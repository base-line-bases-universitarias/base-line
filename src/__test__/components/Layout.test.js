import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';

import Layout from '../../components/Layout';

describe('Esta es una swuit de pruebas', () => {
	test('Debe hacer render del componente Layout', () => {
		const component = shallow(<Layout />);
		expect(component.length).toEqual(1);
		expect(component).toMatchSnapshot();
	});
});
