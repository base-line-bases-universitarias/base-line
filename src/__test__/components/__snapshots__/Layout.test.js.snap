// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Esta es una swuit de pruebas Debe hacer render del componente Layout 1`] = `
<Fragment>
  <Header />
  <Footer />
</Fragment>
`;
